tools/recetox_msfinder/recetox_msfinder.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="recetox_msfinder" name="RECETOX MsFinder" version="@TOOL_VERSION@+galaxy0">
+<tool id="recetox_msfinder" name="RECETOX MsFinder" version="@TOOL_VERSION@+galaxy1">
```

Comment:
this is not needed if you bump the TOOL_VERSION

Reply:
updated in [ae59da5](https://github.com/RECETOX/galaxytools/pull/324/commits/ae59da56e3ad0c0277d5dd9ae60d1d2aef73fd94)